examples/byte-type/byte_type.description

```diff
@@ -1,3 +1,3 @@
-//Byte is 8 bit and it is just shorthand for 0|1|2|...|255 in ballerina.
-//Along with general byte array type, there is also a special syntax for defining base64 based array of bytes.
-//This example demonstrates how byte and byte array can be used with ballerina programs.
+// The `byte` type is 8 bit and it is just shorthand for 0|1|2|...|255 in Ballerina.
```

Comment:
I think we need to rephrase this in a better way: "The `byte` type is 8 bit"
Maybe something like "The size of the `byte` type is defined as 8 bits" or something along those lines?

Reply:
and "in Ballerina" seems redundant info right?